accounts/tests.py

```diff
@@ -0,0 +1,28 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+
+# Create your tests here.
```

Comment:
**nitpick:** I assume this comment can be removed

**praise:**  What a chad for writing test in a personal app.

Reply:
Yes it can